src/cluster.c

```diff
@@ -1046,8 +1046,6 @@ void clusterInit(void) {
      * discover the IP address via MEET messages. */
     deriveAnnouncedPorts(&myself->tcp_port, &myself->tls_port, &myself->cport);
 
-    server.cluster->mf_end = 0;
-    server.cluster->mf_slave = NULL;
     resetManualFailover();
```

Comment:
```
void resetManualFailover(void) {
    if (server.cluster->mf_slave) {
        /* We were a master failing over, so we paused clients and related actions.
         * Regardless of the outcome we unpause now to allow traffic again. */
        unpauseActions(PAUSE_DURING_FAILOVER);
    }
    server.cluster->mf_end = 0; /* No manual failover in progress. */
    server.cluster->mf_can_start = 0;
    server.cluster->mf_slave = NULL;
    server.cluster->mf_master_offset = -1;
}
```

Reply:
resetManualFailover performs an action based on the value of mf_slave before nullifying it.
and this is init time, and the struct is allocated with zmalloc (not zeroed), so we can't remove this nullification

Reply:
thanks! i didn't notice that. I tried modifying the PR, which can be merged or closed. I feel like it’s not important now.